packages/redirect-parser/tests/helpers/main.js

```diff
@@ -1,6 +1,6 @@
 import { fileURLToPath } from 'url'
 
-import { parseAllRedirects } from '../../lib/index.js'
+import { parseAllRedirects } from '../../src/index.js'
```

Comment:
I think this file should be deleted and only the typescript version should be checked in